programs/compressed-token/src/instructions/freeze.rs

```diff
@@ -1,32 +1,36 @@
 use account_compression::{program::AccountCompression, utils::constants::CPI_AUTHORITY_PDA_SEED};
 use anchor_lang::prelude::*;
 use anchor_spl::token::Mint;
-use light_system_program::sdk::accounts::{InvokeAccounts, SignerAccounts};
+use light_system_program::{
+    program::LightSystemProgram,
+    sdk::accounts::{InvokeAccounts, SignerAccounts},
+};
+
+use crate::program::LightCompressedToken;
 
 #[derive(Accounts)]
 pub struct FreezeInstruction<'info> {
     /// UNCHECKED: only pays fees.
     #[account(mut)]
     pub fee_payer: Signer<'info>,
-    #[account(address= mint.freeze_authority.unwrap()
+    #[account(constraint= authority.key() == mint.freeze_authority.unwrap()
```

Comment:
what is the difference between these?

Reply:
address is not supported in anchor 0.30.1